### PR TITLE
Fix #1490: Update bitcrusher example for constant AudioParam values

### LINKED ISSUE #1490: Update bitcrusher example for optimized AudioParams
If (when?) #1464 is fixed, we need to update the bitcrusher demo because it uses `AudioParam`'s.

Similarly if #1480 is fixed, we need to handle that as well.

### PR DESCRIPTION
**Pull request by rtoy**
***
<a href="https://pr-preview.s3.amazonaws.com/rtoy/web-audio-api/pull/1550.html" title="Last updated on Mar 30, 2018, 11:18 PM GMT (728c808)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/WebAudio/web-audio-api/1550/fc74f42...rtoy:728c808.html" title="Last updated on Mar 30, 2018, 11:18 PM GMT (728c808)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -10174,8 +10174,22 @@ registerProcessor('BitCrusher', class extends AudioWorkletProcessor {
 	process (inputs, outputs, parameters) {
 		let input = inputs[0];
 		let output = outputs[0];
+
```
Comment (hoch) [inline index.bs]: Nothing wrong with this code, but how it's done defeats the purpose of this feature completely. Not sure if this is "exemplary".

Comment (rtoy) [reply index.bs]: I can certainly write a bunch of extra code to handle the 4 possible scenarios if you want.

Comment (hoch) [reply index.bs]: Using modulo operation can do the trick, I believe.

```js
...
let step = Math.pow(0.5, bitDepth[i % bitDepth.length]);
this._phase += frequencyReduction[i % frequencyReduction.length];
...
```

**Comment by rtoy** [comment]
I won't change the indentation here, but I think since it's in a `<pre>` block, we don't need to use tabs here and can indent with whatever we want and wrap however we want.

**Pull request merged**